.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Describe changes
+
+Clearly and concisely describe what is included in this PR. What is the PR changing?
+
+## What issues are related
+
+Fixes #
+
+Related to #
+
+## Checklist
+
+<!-- You can erase any parts of this checklist that are not applicable to your PR. -->
+
+### Packages Repositories
+
+* [ ] Have you read the [contributing guidelines](https://github.com/ihmeuw-demographics/packageTemplate/wiki#guide-to-r-package-development) for `ihmeuw-demographics` R packages?
+* [ ] Have you successfully run `devtools::check()` locally?
+* [ ] Have you updated or added function (and vignette if applicable) documentation? Did you update the 'man' and 'NAMESPACE' files with `devtools::document()`?
+* [ ] Have you added in tests for the changes included in the PR?
+* [ ] Do the changes follow the `ihmeuw-demographics` [code style](https://github.com/ihmeuw-demographics/packageTemplate/wiki/Code-style-guide)?
+* [ ] Do your changes need to be immediately included in a new build of [`docker-base`](https://github.com/ihmeuw-demographics/docker-base) or [`docker-internal`](https://github.com/ihmeuw-demographics/docker-internal)? If so follow directions in those repositories to rebuild and redeploy the images.
+
+### Other Repositories
+
+* [ ] Did you update the documentation in the `README.md`? Could someone else on our team replicate or use your work?
```

Comment:
Should this be two different points? Also maybe qualify with "if appropriate" since not all PRs will be significant enough to require README updates?